octoprint_simplyprint/websocket/simplyprint.py

```diff
@@ -997,8 +1000,25 @@ def _handle_ai_snapshot(self, eventtime: float) -> float:
         if ai_interval > 0 and self.webcam_stream.webcam_connected:
             img_data = self.webcam_stream.extract_image()
             headers = {"User-Agent": "Mozilla/5.0"}
-            data = json.dumps({"api_key": self.settings.get(["printer_token"]), "image_array": img_data, "interval": ai_interval}).encode('utf8')
+            data = json.dumps(
+                {
+                    "api_key": self.settings.get(["printer_token"]),
+                    "image_array": img_data,
+                    "interval": ai_interval,
+                    "printer_id" : 'ahe728nh28jeh2983ne', #unique id for the printer. No two printer_id's should be the same.
```

Comment:
printer_id should not be hard-coded, needs to be unique and should be retrieved via self.settings.get.